app/assets/javascripts/index.js

```diff
@@ -0,0 +1,235 @@
+		popupAnchor: [1, -34],
+		shadowSize: [41, 41]
+	});
+	const yellowIcon = new L.Icon({
+		iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-yellow.png',
+		shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+		iconSize: [25, 41],
+		iconAnchor: [12, 41],
+		popupAnchor: [1, -34],
+		shadowSize: [41, 41]
+	});
+
+	let birdData = null;
+	let birdImages = null;
+	const mymap = L.map('mapid').setView([37.42, -121.91], 13);
+	L.tileLayer('https://a.tiles.mapbox.com/styles/v1/lohneswright/ciocejooj006obdnjhmd2x9qp/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibG9obmVzd3JpZ2h0IiwiYSI6IngzbVlqNnMifQ.OwxjrBKoGXc60NB5x6GKzw', {
+		maxZoom: 22
+	}).addTo(mymap);
+
+	const legend = L.control({
+		position: 'topright'
+	});
+	legend.onAdd = function(map) {
+		const div = L.DomUtil.create('div', 'legend');
+		div.innerHTML += '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png" height=41 width=25>Your Location<br>';
+		div.innerHTML += '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-gold.png" height=41 width=25>Bird-Watching Park<br>';
+		div.innerHTML += '<img src="https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-blue.png" height=41 width=25>Recent Bird Sighting';
+		div.style.background = 'silver';
+		return div;
+	};
+	legend.addTo(mymap);
+
+	function setContent(birddata, imgsrc, elem) {
+		$(elem).empty();
+		$(elem).append(
+			`
+      <img class="card-img-top" src="${imgsrc}">
+      <div class="card-body">
+        <h5 class="card-title">${birddata.comName}</h5>
+        <h6 class="card-subtitle mb-2 text-muted">${birddata.sciName}</h6>
+        <p class="card-text">${birddata.distTo} miles away, at ${birddata.locName}</p>
+        <p class="card-text">reported ${moment(birddata.obsDt).fromNow()}</p>
+      </div>
+      <a href="https://ebird.org/species/${birddata.speciesCode}" class="btn btn-primary">View On Ebird</a>
+      `
+		);
+	}
+
+
+	function setContentLoc(i, birddata, imgsrc, elem) {
+		$(elem)[i].innerHTML
+      += `
+        <center>
+        <img class="card-img-top" src="${imgsrc}" style="width: 10rem; height: 10rem;">
+        </center>
+        <div class="card-body" style="padding: 0">
+          <a href="https://ebird.org/species/${birddata.speciesCode}"> <h5 class="card-title">${birddata.comName}</h5> </a>
+          <h6 class="card-subtitle mb-2 text-muted">${birddata.sciName}</h6>
+          <h6> <p>Last Seen: ${moment(birddata.obsDt).fromNow()}</p></h6>
+        </div>
+      `;
+	}
+
+	function setMarker(birddata, index) {
+		const marker = new L.Marker(
+			L.latLng(birddata.lat, birddata.lng),
+			{
+				name: `bird_marker_${index}`,
+				icon: blueIcon
+			}
+		);
+		mymap.addLayer(marker);
+		marker.bindPopup(`<b>${birddata.comName}</b><br/>${birddata.sciName}.`).openPopup();

```

Comment:
b is deprecated, use strong instead (or a heading element if more appropriate).
Though I'm not sure if you're still going to use these bird markers going forward?